content/deploy/install/services.md

```diff
@@ -267,6 +268,34 @@ After installing all other services, install the UI with these steps:
 If the install is successful, the UI is available on its
 [default port]({{< ref "default-ports" >}}).
 
+## Audit Trail Service [Optional]
+
+
+The Rhize Audit service provides audit trail for database changes to install. The Audit service uses InfluxDB2 for storage.
+
+Install Audit Service with these steps:
+
+1. Modify the InfluxDB Helm YAML file as needed. It is *recommended* to set the admin password and token in the helm yaml file to prevent over writing the values with random values every deploy.
```

Comment:
```suggestion
1. Modify the InfluxDB Helm YAML file as needed. It is *recommended* to set the admin password and token in the Helm YAML file to prevent over writing the values with random values every deploy.
```